warehouse/query-core/src/main/java/datawave/query/predicate/TLDEventDataFilter.java

```diff
@@ -618,6 +618,23 @@ private void extractQueryFieldsFromScript(ASTJexlScript script) {
         queryFields = Collections.unmodifiableList(queryFields);
     }
     
+    /**
+     * Fields are always uppercase
+     * 
+     * @param field
+     *            the field
+     * @return true if all characters are uppercase
+     */
+    private boolean containsLowers(String field) {
+        for (char c : field.toCharArray()) {
```

Comment:
This is hacky.  Lets use that QueryFieldVisitor that we created a year ago....

Reply:
Opting to use the configured fields in the QueryIterator per our discussion